### PR TITLE
fcitx5-configtool, fcitx5-gtk, fcitx5-qt

### PR DESCRIPTION
**Pull request by amuramatsu**
for #16425

### CODE REVIEW

**Comment by xtkoba** [comment]
This is not a proper review.

Are you going to enable GIR in `fcitx5-gtk-common` package? I really want someone who can inherit my cross-GIR techniques.

**Comment by amuramatsu** [comment]
Sorry for the late reply

> Are you going to enable GIR in fcitx5-gtk-common package? I really want someone who can inherit my cross-GIR techniques.

It is interesting, but I cannot take a time for it now...

File: x11-packages/fcitx5-gtk-common/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=https://fcitx-im.org/
+TERMUX_PKG_DESCRIPTION="Fcitx gtk library"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=5.0.23
+TERMUX_PKG_SRCURL=https://github.com/fcitx/fcitx5-gtk/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=4f00c0125eab89e6a4f100a00540a73afb17db02108d1a7d3e73823da6fa577d
+TERMUX_PKG_DEPENDS="fcitx5, libxkbcommon, glib, libx11"
+#TERMUX_PKG_BUILD_DEPENDS="g-ir-scanner"
+#TERMUX_PKG_DISABLE_GIR=false
```
Comment (xtkoba) [inline x11-packages/fcitx5-gtk-common/build.sh:10]: Remove these lines.

File: x11-packages/fcitx5-gtk-common/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DENABLE_GIR=OFF
+-DENABLE_GTK2_IM_MODULE=ON
+-DENABLE_GTK3_IM_MODULE=ON
+-DENABLE_GTK4_IM_MODULE=ON
+"
+
+#termux_step_pre_configure() {
+#	termux_setup_gir
+#}
```
Comment (xtkoba) [inline x11-packages/fcitx5-gtk-common/build.sh:20]: Remove these lines.

**Review by xtkoba — COMMENTED** [review]
No problem. But then please remove cross-GIR stuff from `build.sh`. Thanks.

**Pull request merged**